core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2507,6 +2515,15 @@ public String toString() {
               + "an exponential backoff. This property determines the maximum duration to retry for"
               + " before giving up.")
           .build();
+  public static final PropertyKey USER_RPC_RETRY_LARGE_NUM_RETRY =
```

Comment:
why PropertyKey `USER_RPC_RETRY_LARGE_NUM_RETRY` has the real name `Name.USER_RPC_RETRY_MAX_NUM_RETRY` which is different?  in addition, I could not find how `PropertyKey.USER_RPC_RETRY_LARGE_NUM_RETRY` is used

Reply:
Plus, if possible, I would try not to add new configuration properties

Reply:
Put it as a static variable in the metaMasterSync instead of a property key, thanks